sdk/src/types/block/output/mod.rs

```diff
@@ -224,6 +227,21 @@ impl Output {
         creation_index: SlotIndex,
         target_index: SlotIndex,
     ) -> Result<u64, Error> {
+        let decayed_mana = self.decayed_stored_and_potential_mana(protocol_parameters, creation_index, target_index)?;
+        
+        decayed_mana
+            .stored
+            .checked_add(decayed_mana.potential)
+            .ok_or(Error::ConsumedManaOverflow)
+    }
+
+    /// Returns the decayed stored and potential mana of the output.
+    pub fn decayed_stored_and_potential_mana(
```

Comment:
Nit but I'd just call this `decayed_mana` since it's the type name. Just resolve if you don't want it, I don't feel strongly.

Reply:
I think that makes sense, initially I was about to return `(u64, u64)` so the name would then help to know what value is what, but then I changed to struct so it's really clear